sdk/dynatrace/Azure.ResourceManager.Dynatrace/src/Generated/DynatraceMonitorResource.cs

```diff
@@ -622,7 +622,7 @@ public virtual Pageable<DynatraceMonitoredResourceDetails> GetMonitoredResources
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>Monitors_GetVmHostPayload</description>
+        /// <description>Monitors_GetVMHostPayload</description>
```

Comment:
it's `VM`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/dynatrace.json#L100